metroae

```diff
@@ -2517,7 +2517,7 @@ function copy_ssh_id {
 
     sshpass=" "
     if [[ ! -z $SSHPASS_PASSWORD ]]; then
-       sshpass="sshpass -p'$SSHPASS_PASSWORD'"
+       sshpass="sshpass -p$SSHPASS_PASSWORD"
```

Comment:
This is fail to allow to special characters in password correct. I thought the single quotes were added for that purpose

Reply:
I checked with a special case password, and without any single quotes, this will work.